### ISSUE TITLE
Can't TP to Exodor Locations

### ISSUE THREAD

**Issue by ap0521** [Owyn/infinity-journal#3]
Gives error "Zone 2000 not found in Village Atlas" when attempting to move to a location within Exodor.

**Comment by Owyn** [comment]
Can you move to Exodor with Village Atlas?

**Comment by ap0521** [comment]
No, you cannot move directly to Exodor with the atlas, though the default travel journal is able to properly move you to and around Exodor, so there should be a way to fix it.

**Comment by Owyn** [comment]
That'd require a single saved journal entry (real one) inside exodor for module to use instead of using village Atlas, and then change coordinates on arrival,

so yea - it can be done

thought, I don't play Tera anymore and can't even test it even if I were to write anything,

so it'd be a nice idea to ask someone to do it

**Comment by MrAutismo** [comment]
Why do u even need the village atlas stuff for travel journal?
